documentation/asciidoc/computers/raspberry-pi/bcm2711-bootloader.adoc

```diff
@@ -559,6 +559,33 @@ This option may be set to 0 to block self-update without requiring the EEPROM co
 
 Default: `1`
 
+==== `secure-boot` configuration properties in `config.txt`
+The following `config.txt` properties are used to program the `secure-boot` OTP settings. These changes are irreversible and can only be programmed via `RPIBOOT` when flashing the bootloader EEPROM image.  This ensures that `secure-boot` cannot be set remotely or via accidentally inserting a stale SD card image.
```

Comment:
This should probably link to the `RPIBOOT` part of the documentation?
"or via accidentally" -> "or by accidentally"

Reply:
Added a link to the USBBOOT repo and secure-boot tutorial